website/www/site/content/en/documentation/runtime/environments.md

```diff
@@ -116,8 +116,8 @@ By default, no licenses/notices are added to the docker images.
 
 To examine the containers that you built, run `docker images` from anywhere in the command line. If you successfully built all of the container images, the command prints a table like the following:
 ```
-REPOSITORY                          TAG                 IMAGE ID            CREATED           SIZE
-apache/beam_java_sdk               latest              16ca619d489e        2 weeks ago        550MB
+REPOSITORY                         TAG                 IMAGE ID            CREATED           SIZE
+apache/beam_java_8sdk              latest              16ca619d489e        2 weeks ago        550MB
```

Comment:
typo here:
```suggestion
apache/beam_java8_sdk              latest              16ca619d489e        2 weeks ago        550MB
```

I'm assuming this change and the others like it are the result of a search to find and update all the references to `beam_java_sdk`, so we don't need to worry about there being other references?

Reply:
Let's make sure that the correct container name is set in the environment send to the remote SDKs that use Java transforms as cross-language transforms. (We don't have integration tests setup for this by we can just try existing Kafka/SQL x-lang examples to confirm)

https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/kafkataxi
https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/sql_taxi.py

Reply:
We should make sure the references to `beam_java_sdk` are updated in those examples too

Reply:
Weird! I don't know why they didn't show up in my initial ctrl-f

Reply:
I bet it was a race condition, those examples were added in just the last couple of months

Reply:
I went ahead and ran the Kafka example "from local Beam repo" without specifying the --sdk_harness_container_image_overrides to see if we should pass the correct URL, which it seems to do (based on the worker container manifest):
```
{
      "args": [ ... ],
      "image": "apache/beam_java8_sdk:2.25.0.dev",
      "imagePullPolicy": "IfNotPresent",
      "name": "sdk-1-0",
      "volumeMounts": [ {
        "mountPath": "/var/opt/google",
        "name": "persist"
},     
```

Dataflow can't actually grab that container because it doesn't exist (on hub.docker.com), so the job itself doesn't work, but it seems to be grabbing the right URL.

Reply:
That looks correct. Thanks.